src/parser/parser.py

```diff
@@ -1,37 +1,110 @@
+            try:
+                value = kv3_to_dict(value)
+                if value is None:
+                    continue # Continue to next value if its not serializable
+            except TypeError:
+                return None
+
+        dict[key] = value
+
+    return dict
+
+
+# Converts kv3 object to dict, then writes dict to json
+def kv3_to_json(kv3_obj, output_file):
+    # output_file should always end in .json
+    if not output_file.endswith('.json'):
+        raise ValueError('output_file must end in .json')
+    
+    return json_utils.write(output_file, kv3_to_dict(kv3_obj))
+
+# Default method to load vdata files
+def _load_vdata_default(vdata_path):
+    print('Starting vdata_path:', vdata_path)
+    data = kv3.read(vdata_path+'.vdata')
+    out_path = vdata_path.replace('/vdata','')+'.json'
+    print('Finished out_path:', out_path)
+    kv3_to_json(data, out_path)
+    
+
+    return data
+
+
 
 class Parser:
     def __init__(self, language='english'):
         # constants
-        self.DATA_DIR = './decompiled-data/'
+        self.DATA_DIR = './decompiler/decompiled-data/'
+        self.DATA_VDATA_DIR = self.DATA_DIR+'vdata/'
         self.language = language
 
         self._load_vdata()
         self._load_localizations()
 
     def _load_vdata(self):
-        generic_data_path = os.path.join(self.DATA_DIR, 'scripts/generic_data.vdata')
-        self.generic_data = kv3.read(generic_data_path)
+        # Convert .vdata_c to .vdata and .json
+        scripts_path = 'scripts'
+
+        #non_standard_loads = ['abilities']
+
+        #excepted_loads = ['misc']
+
+        path_to_load = self.DATA_VDATA_DIR+scripts_path
+        for file in os.listdir(path_to_load):
+            if file.endswith('.vdata_c'):
+                os.rename(file, file[:-2])
+
+        # Generic
+        self.generic_data = _load_vdata_default(self.DATA_VDATA_DIR+scripts_path+'/generic_data')
+
+        # Hero
+        self.hero_data = _load_vdata_default(self.DATA_VDATA_DIR+scripts_path+'/heroes')
 
-        hero_data_path = os.path.join(self.DATA_DIR, 'scripts/heroes.vdata')
-        self.hero_data = kv3.read(hero_data_path)
+        # Abilities
+        abilities_subpath = os.path.join(scripts_path, 'abilities')
+        abilities_data_path = os.path.join(self.DATA_VDATA_DIR, abilities_subpath+'.vdata')
+        with open(abilities_data_path, 'r') as f:
+            content = f.read()
+            # replace 'subclass:' with ''
+            # subclass features in kv3 don't seem to be supported in the keyvalues3 python library
+            content = content.replace('subclass:', '')
+            # write new content to tempfilex
+            with tempfile.NamedTemporaryFile(mode='w', delete=False) as tf:
+                tf.write(content)
+                self.abilities_data = kv3.read(tf.name)
+                kv3_to_json(self.abilities_data, os.path.join(self.DATA_DIR, abilities_subpath+'.json'))
```

Comment:
some of these changes look like they came from an old version of parser. As this logic now lives in `remove-subclass.py`